HackIllinois/ViewControllers/HILeaderboardViewController.swift

```diff
@@ -65,20 +63,7 @@ extension HILeaderboardViewController {
 extension HILeaderboardViewController {
```

Comment:
Can you remove the any `HIErrorViewDelegate` code from `HILeaderboardViewController` since the Leaderboard View will no longer need it?